libs/components/modals/src/lib/modules/confirm/confirm.service.ts

```diff
@@ -21,16 +22,22 @@ import { SkyConfirmComponent } from './confirm.component';
 })
 export class SkyConfirmService {
   #modalService: SkyModalService;
+  #logService: SkyLogService;
 
-  constructor(modalService: SkyModalService) {
+  constructor(modalService: SkyModalService, logService: SkyLogService) {
     this.#modalService = modalService;
+    this.#logService = logService;
   }
 
   /**
    * Opens a dialog using the specified options.
    * @param config Specifies configuration options for the dialog.
    */
   public open(config: SkyConfirmConfig): SkyConfirmInstance {
+    if (config.buttons?.some((button) => button.autofocus != undefined)) {
+      this.#logService.deprecated('autofocus');
```

Comment:
Is using the log service part of the requirements for this story? What's wrong with just adding `@deprecated` to the property and leaving this bit out?

Reply:
removed